### DISCUSSION TITLE
Add "Recently Viewed" as a Product Query Source option in the Loop Grid and Loop Carousel widgets

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21292] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Adding **recently viewed** query for Loop Grid / Loop Carousel widget would be great. This will be really helpful.

### Describe the solution you'd like

🔥Add the recently viewed query to the Loop Grid and Loop Carousel widgets.

**Comment by bobhuf** [comment]
I'd like to see this query type added as well. Currently using the native "Recently viewed products" widget but it requires enabling widgets and the html output is bad.

**Comment by phkoon** [comment]
Come on Elementor, it's been more than 1 year, basic feature for e-Commerce....

**Comment by loucasl** [comment]
I don't even think Crocoblock has this query type do they?
